src/components/ResizeHandler/ResizeHandler.js

```diff
@@ -15,6 +15,13 @@ const propTypes = {
   maxWidthBrkpts: PropTypes.objectOf(PropTypes.number),
   stylePropHeight: PropTypes.string,
   onResize: PropTypes.func,
+
+  // This property is used to re-apply the classes, for example when the className of the observed node changes.
+  forceUpdate: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+    PropTypes.bool,
+  ]),
```

Comment:
Is that really needed ?

Reply:
I use it to update the IABP doc when changing disabled elements.